reviewrot/__init__.py

```diff
@@ -104,6 +104,9 @@ def get_arguments(cli_arguments, config_arguments, choices):
             elif 'cacert' in config_arguments:
                 # expand ~, environment variables, etc if it's a path
                 parsed_arguments['ssl_verify'] = config_arguments.get('cacert')
+            else:
```

Comment:
Please considering disabling the specific warnings instead of forcing SSL verify to false.
Tips: https://stackoverflow.com/questions/27981545/suppress-insecurerequestwarning-unverified-https-request-is-being-made-in-pytho

Reply:
but the problem is in line: 
`gl = gitlab.Gitlab(host, token, ssl_verify=ssl_verify)`
The value of ssl_verify ended up being None, which resulted in error, so the default should be False as is when specifying arguments through command line.

Reply:
In testing this on my own system, I discovered another problem with the way the current code is handling SSL verification: the way this function (`get_arguments()`) is being called from `bin/review-rot`, the config file arguments are *never* passed to it; `main()` supplies `config.get('arguments')`, but the config file arguments are stored at the top level of the dict, not under the 'arguments' key.

`review-rot` also fails to use the CA cert supplied as a command line argument at all.

Regardless of when and how these issues were introduced, they should probably be addressed now.